testing/entrypoints/__init__.py

```diff
@@ -0,0 +1,285 @@
+import importlib
+from pathlib import Path
+import pytest
+from click import BaseCommand
+from click.testing import CliRunner
+from typing import Callable, Iterable, Any, Mapping, Optional, List, Union
+import inspect
+from dataclasses import dataclass
+import sys
+import shlex
+import os
+import shutil
+from unittest.mock import patch
+
+
+class ArgsExpander:
+    def __init__(self, orig_func: Callable):
+        self.orig_func = orig_func
+
+    def normalize(self, args: Iterable[Any], kwargs: Mapping[str, Any]) -> dict:

```

Comment:
should we also keep the docstring structure in the tests also coherent with the one in all Gandlf files? If so, this should be edited

Reply:
Thanks folks! Sure. Updated docstrings to the common format.